README.md

```diff
@@ -25,6 +25,12 @@ mkdir myApp && cd myApp
 
 # Run the generator
 yo react-zeal
+
+# install all the things
+npm install
+
+# start your app on http://localhost:3000
+npm start
```

Comment:
We use Yarn in our generated projects, so this should probably be `yarn` and `yarn start`.  It might also be worth mentioning that we use Yarn in our generated projects to make that clearer.

The line above about `npm i -g` might also want to change to `yarn global add`.